tcms/issuetracker/base.py

```diff
@@ -15,9 +15,13 @@ class IntegrationThread(threading.Thread):
     def __init__(self, rpc, bug_system, execution, bug_id):
         """
             :param rpc: Bug tracker RPC object
+            :type rpc: dict
```

Comment:
This is object, not a dict. Actual type is different for each integration class. That's the object facilitating communication with the remote system.

Reply:
So what should I say the type is here then?

Reply:
This is of type `object` in the base class and then a specific type coming from a 3rd party library in the actual class providing the implementation.